tools/src/wyvern/tools/lexing/WyvernLexer.x

```diff
@@ -253,7 +253,7 @@ import wyvern.tools.errors.ToolError;
     terminal Token cSquareBracket_t ::= /\]/ {: RESULT = token(RBRACK,lexeme); :};
 
  	terminal Token shortString_t ::= /(('([^'\n]|\\.|\\O[0-7])*')|("([^"\n]|\\.|\\O[0-7])*"))|(('([^']|\\.)*')|("([^"]|\\.)*"))/ {:
- 		RESULT = token(STRING_LITERAL,lexeme.substring(1,lexeme.length()-1));
+ 		RESULT = token(STRING_LITERAL,lexeme.substring(1,lexeme.length()-1).replace("\\n", "\n"));
```

Comment:
Hi Jonathan,

Using the Properties library or the StringTokenizer library causes the following two test cases to fail:

1. wyvern.tools.tests.StdlibTests.testRegex()
2. wyvern.tools.tests.StdlibTests.testRegexLexer()

The reason for this is that if a string literal contains "\\<some character>" (for e.g., "\\d"), then it is parsed as "<some character>" ("d" in our example).

In the case of the Properties library, this happens because a single backslash character indicates that you want to break a long single-line property value into multiple lines. Check this page for details: https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html

I haven't been able to pinpoint the exact reason why the StreamTokenizer library behaves in this way.

Thanks,
Selva